applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.cpp

```diff
@@ -294,7 +419,7 @@ double MassConservationCheckProcess::ComputeInterfaceArea(){
 
             if ( rGeom.PointsNumber() == 3 ){ p_modified_sh_func = Kratos::make_unique<Triangle2D3ModifiedShapeFunctions>(it_elem->pGetGeometry(), Distance); }
             else if ( rGeom.PointsNumber() == 4 ){ p_modified_sh_func = Kratos::make_unique<Tetrahedra3D4ModifiedShapeFunctions>(it_elem->pGetGeometry(), Distance); }
-            else { KRATOS_ERROR << "The process can not be applied on this kind of element" << std::endl; }
+            else { KRATOS_ERROR << "The utility can not be applied on this kind of element" << std::endl; }
```

Comment:
Same comment. This should be moved to an auxiliary method that returns the pointer.

Reply:
done